src/components/components/Component.js

```diff
@@ -9,9 +9,48 @@ import Events from '../../lib/Events';
 import Select from 'react-select';
 import { updateEntity } from '../../lib/entity';
 import axios from 'axios';
+import "./w3.css";
 
 const isSingleProperty = AFRAME.schema.isSingleProperty;
 
+class GltfPopUp extends React.Component {
+  static propTypes = {
+  };
+
+  constructor(props) {
+    super(props);
+  }
+
+  closeModal() {
+    this.props.closePopup();
+  }
+
+  componentDidMount() {
+    this.closeModal = this.closeModal.bind(this);
+  }
+
+  render() {
+    var iframeLink = "http://localhost:3000/admin/scene/" + this.props.sceneId + "/object/" + this.props.objectId;
```

Comment:
Can you make this into a REACT env variable and add it to the `env` file? Like the initial domain part of it `http://localhost:3000`